tests/e2e/edgesite/edgesite_test.go

```diff
@@ -104,70 +121,77 @@ var _ = Describe("Application deployment test in E2E scenario using EdgeSite", f
 			// Start test timer
 			testTimer = DeploymentTestTimerGroup.NewTestTimer(testSpecReport.LeafNodeText)
 		})
+
 		AfterEach(func() {
 			// End test timer
 			testTimer.End()
 			// Print result
 			testTimer.PrintResult()
-			var podlist metav1.PodList
-			label := nodeName
-			podlist, err := utils.GetPods(ctx.Cfg.K8SMasterForKubeEdge+constants.AppHandler, label)
+
+			labelSelector := labels.SelectorFromSet(map[string]string{"kubeedge": "e2e-test"})
```

Comment:
only some tests use "e2e-test" label, but not all

Reply:
for pod test, we may create multi pod and we use this label for the pod that is created by e2e and after each case finished, we use the label to clean the pod.